turbinia/lib/libcloudforensics.py

```diff
@@ -143,17 +147,29 @@ def list_instances(self):
     Returns:
       Dictionary with name and metadata for each instance.
     """
-    operation = self.gce_api().instances().aggregatedList(
-        project=self.project_id).execute()
-    result = self.gce_operation(operation, zone=self.default_zone)
+    have_all = False
```

Comment:
have all what?
should this be have_all_tokens?

Reply:
Changed to have_all_tokens.